fastlane/Fastfile

```diff
@@ -29,8 +29,7 @@ platform :ios do
       device: "iPhone SE (3rd generation)",
       result_bundle: true,
       scheme: options[:scheme],
-      testplan: options[:testplan],
-      prelaunch_simulator: true
```

Comment:
Is removing the `prelaunch_simulator` part of making the pipeline faster?

Reply:
It didn't seem to be making any difference so removed it